bin/github-review.sh

```diff
@@ -45,12 +45,21 @@ if [[ -z $chainid ]]; then
     echo "ERROR: Chain ID not specified as per the PR Template" 1>&2
     exit 1
 fi
-chainInfo="https://raw.githubusercontent.com/ethereum-lists/chains/refs/heads/master/_data/chains/eip155-$chainid.json"
-rpc="$(curl -sfL "$chainInfo" | jq -r '.rpc[0]')"
+
+# Fetch RPC from DefiLlama's chainlist
+chainlist="$(curl -sfL 'https://chainlist.org/rpcs.json')"
+if [[ -z $chainlist ]]; then
+    echo "ERROR: Failed to fetch chainlist from DefiLlama" 1>&2
+    exit 1
+fi
+
+# Extract RPC for the specified chain ID using jq
+rpc="$(echo "$chainlist" | jq -r ".[] | select(.chainId == $chainid) | .rpc[0].url")"
```

Comment:
If this command fails, the scripts exits with an error code `22`. See https://github.com/safe-global/safe-deployments/actions/runs/16163105042/job/45618604201 for reference. 

So, script will never report `ERROR: RPC not found for chain ID $chainid in DefiLlama's chainlist` making it difficult to find root cause of failure. I suggest we fix the command in this PR.

Reply:
Oops, its `//`. And I just checked, and there is no error:

- the `.[]` splat will create a stream of JSON objects
- the `select` will filter out only objects that match the condition (i.e. same chain ID)
- the `.rpc[0].url` application only applies to matched objects in the stream. If there are none (no chain ID), then it won't be applied.

This means that if the chain isn't there - the code as provided here works. You can check this with:

```
$ curl -sfL 'https://chainlist.org/rpcs.json' | jq -r ".[] | select(.chainId == 982371)"
$ echo $?
0
```

(where `982371` doesn't exist).

Reply:
> the scripts exits with an error code 22.

I just checked, and this is a `curl` exit code for HTTP error when using the `-f` flag, and unrelated to the `jq` command.